tests/test_global_mutual_information_loss.py

```diff
@@ -116,24 +117,35 @@ def transformation(translate_params=(0.0, 0.0, 0.0), rotate_params=(0.0, 0.0, 0.
 
 class TestGlobalMutualInformationLossIll(unittest.TestCase):
 
-    def test_ill_shape(self):
+    @parameterized.expand(
+        [
+            ("mismatched_simple_dims", torch.ones((1, 2), dtype=torch.float), torch.ones((1, 3), dtype=torch.float)),
+            (
+                "mismatched_advanced_dims",
+                torch.ones((1, 3, 3), dtype=torch.float),
+                torch.ones((1, 3), dtype=torch.float),
+            ),
+            # You can add more test cases as needed
+        ]
+    )
+    def test_ill_shape(self, name, input1, input2):
```

Comment:
For all the name in this PR, can we just leave it as a comment line instead of an argument?
Something like this:https://github.com/Project-MONAI/MONAI/blob/dev/tests/test_compute_meandice.py#L213